node/src/inherent_data.rs

```diff
@@ -23,8 +23,6 @@ use sp_timestamp::InherentDataProvider as TimestampIDP;
 use sp_timestamp::Timestamp;
 use std::error::Error;
 use time_source::TimeSource;
-
-#[cfg(feature = "block-beneficiary")]
 use {sidechain_runtime::BeneficiaryId, sp_block_rewards::BlockBeneficiaryInherentProvider};
```

Comment:
Please merge them into other imports. There is no need to leave a separate list, in curly braces, after feature was removed.

Reply:
merged